Makefile

```diff
@@ -13,17 +13,13 @@ package:
 	-v ${PWD}:/go/src/github.com/CMSgov/bcda-app packaging $(version)
 
 lint:
-	docker-compose -f docker-compose.test.yml run --rm tests golangci-lint run --deadline=3m --skip-dirs=ssas
-	docker-compose -f docker-compose.test.yml run --rm tests gosec -exclude-dir=ssas ./...
+	docker-compose -f docker-compose.test.yml run --rm tests golangci-lint run --deadline=3m
+	docker-compose -f docker-compose.test.yml run --rm tests gosec ./...
 
-lint-ssas:
```

Comment:
Does the `.PHONY` section need to be updated with the removal of some targets?

Reply:
It seems the SSAS-related targets never got put in the .PHONY section, so instead of being removed others need to be added.  I've done so in the latest commit, and sorted the `.PHONY` section alphabetically because it's getting a bit unwieldy.